doc/release-notes.md

```
@@ -43,7 +43,7 @@ Downgrading to a version older than 0.14.0.3 is no longer supported due to
 changes in the "evodb" database format. If you need to use an older version,
 you must either reindex or re-sync the whole chain.
 
-### Downgrade of masternodes to < 0.17.0.2
+### Downgrade of masternodes to < 0.17.0.3
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)